crates/autopilot/src/database/onchain_order_events.rs

```diff
@@ -613,21 +606,19 @@ fn convert_onchain_order_placement(
         false
     };
 
-    let liquidity_owner = if liquidity_order_owners.contains(&owner) {
+    // remove liquidity owners as part of the https://github.com/cowprotocol/infrastructure/issues/1032
+    // leaving uncommmented for compiler to catch
+    let _liquidity_owner = if liquidity_order_owners.contains(&owner) {
```

Comment:
tracked https://github.com/cowprotocol/services/issues/2367

Reply:
Why can't we already remove this block now (what is the compiler supposed to catch)?